ViewGenerator/contentFiles/global.d.ts

```diff
@@ -54,6 +54,7 @@ declare module "ResourceLoader" {
     export type ResourceLoaderUrlFormatter = (resourceKey: string, ...params: string[]) => string;
 
     export const ResourceLoader: React.Context<ResourceLoaderUrlFormatter>;
+    export function createResourceLoaderUrlFormatter(customResourceBaseUrl: string, viewName: string): ResourceLoaderUrlFormatter;
```

Comment:
customResourceBaseUrl ... shouldn't be exposed

Reply:
Actually expose a function that returns the url

```
export function formatUrl(viewName: string, resourceKey: string, ...params: string[]): string{
 const urlTail = [resourceKey].concat(params).map(p => encodeURIComponent(p)).join("&");
 return `${customResourceBaseUrl}/${viewName}/?${urlTail}`;
}
```

Reply:
Store the customResourceBaseUrl in a local variable that should be set on the Loader (using the url params already grabbed)

Reply:
Done